src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -134,12 +134,19 @@
       <AdditionalProperties>TargetFramework=net46</AdditionalProperties>
     </ProjectReference>
     <PackageReference Include="EnvDTE" Version="$(EnvDTEVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Managed" Version="$(MicrosoftVisualStudioProjectSystemManagedVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" Version="$(MicrosoftVisualStudioShell150Version)" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.8.0" Version="$(MicrosoftVisualStudioShellInterop80Version)" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.9.0" Version="$(MicrosoftVisualStudioShellInterop90Version)" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.10.0" Version="$(MicrosoftVisualStudioShellInterop100Version)" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime" Version="$(MicrosoftVisualStudioShellInterop153DesignTimeVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Framework" Version="$(MicrosoftVisualStudioShellFrameworkVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Utilities" Version="$(MicrosoftVisualStudioUtilitiesVersion)" />
+    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
```

Comment:
Does this cause specific versions of MSBuild packages to be pulled into the CompilerExtension?

Reply:
This is to resolve nuget dependencies.  `Microsoft.VisualStudio.ProjectSystem.Managed` brings in older versions of the MSBuild API than are used elsewhere in `Roslyn.sln`.  Adding these packages unifies the version, but we don't deploy these binaries as part of VSIX deploy.  We will use whatever version of MSBuild that devenv.exe finds (provided it is newer than 15.3).

Reply:
Great! Just wanted to check.

Reply:
@jmarolf Can we add a comment? My worry is we use CompilerExtension.csproj to determine the potential set of framework facades we need to include in PortableFacades.swr; so this could have accidental downstream impacts.

> but we don't deploy these binaries as part of VSIX deploy

I would have imagined this was infact happening. Why isn't it?

Reply:
`NuGetPackageToIncludeInVsix` is what is used to track what is in `PortableFacades.swr` correct? My understanding was that is was an explicit operation, not implicit so this wouldn't happen.  This is what gets included in the VSIX when I build:

![image](https://user-images.githubusercontent.com/9797472/39728533-6393c67c-520c-11e8-8ee2-602236e46d6d.png)